a10_octavia/controller/worker/flows/vthunder_flows.py

```diff
@@ -187,6 +187,10 @@ def _get_create_amp_for_lb_subflow(self, prefix, role):
             name=sf_name + '-' + a10constants.VTHUNDER_BY_LB,
             requires=constants.LOADBALANCER,
             provides=a10constants.VTHUNDER))
+        create_amp_for_lb_subflow.add(
+            vthunder_tasks.VThunderComputeConnectivityWait(
+                name=a10constants.MASTER_CONNECTIVITY_WAIT,
```

Comment:
Should this just be `a10constants.VTHUNDER_CONNECTIVITY_WAIT` we don't know the role at this point

Reply:
Done